<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GraphicsView</class>
 <widget class="QWidget" name="GraphicsView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1127</width>
    <height>690</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget#GraphicsView{
background-color: rgb(255, 255, 255);
}

QToolButton{
font: 14pt &quot;楷体&quot;;
color: rgb(255, 255, 255);
border:2px solid  rgb(0, 125, 165);
border-radius:10px;
background-color: rgb(0, 125, 165);
}
QLabel{
font: 14pt &quot;楷体&quot;;
border-radius:1px;
}
QGraphicsView{
color: rgb(255, 255, 255);
border:1px solid  rgb(0, 125, 165);
}
/*
QScrollBar:vertical{
width:30px;
}
QScrollBar:horizontal{
width:30px;
}*/

QTableWidget{
font: 14pt &quot;楷体&quot;;
gridline-color: rgb(0, 0, 0);
border:1px solid rgb(0, 122, 165);
border-radius:5px;
background-color: rgb(255, 255, 255);
}
QTableWidgetItem{
border:1px solid rgb(0, 0, 0);
}

QGroupBox{
font: 16pt &quot;楷体&quot;;
color:rgb(0, 125, 165);
color:rgb(0,   0,   0);
border:1px none rgb(0, 125, 165);
border-radius:5px;
margin-top:20px;
}
QGroupBox:title{
subcontrol-origin: margin;
left: 5px;
}</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" stretch="1,10,1">
   <property name="spacing">
    <number>5</number>
   </property>
   <property name="leftMargin">
    <number>5</number>
   </property>
   <property name="topMargin">
    <number>5</number>
   </property>
   <property name="rightMargin">
    <number>5</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_8">
     <property name="spacing">
      <number>5</number>
     </property>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_6" stretch="1,1,1,1">
       <property name="spacing">
        <number>30</number>
       </property>
       <property name="leftMargin">
        <number>10</number>
       </property>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <property name="spacing">
          <number>5</number>
         </property>
         <item>
          <widget class="QLabel" name="lbNormal">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>40</width>
             <height>60</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>40</width>
             <height>60</height>
            </size>
           </property>
           <property name="text">
            <string>正常
状态</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="lbImageNormal">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>70</width>
             <height>60</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>70</width>
             <height>60</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">border-image: url(:/Image/normal.png);</string>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <property name="spacing">
          <number>5</number>
         </property>
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QLabel" name="lbOffLine">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>40</width>
             <height>60</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>40</width>
             <height>60</height>
            </size>
           </property>
           <property name="text">
            <string>掉线
状态</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="lbImageOffLine">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>70</width>
             <height>60</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>70</width>
             <height>60</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">border-image: url(:/Image/offLine.png);</string>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <property name="spacing">
          <number>5</number>
         </property>
         <item>
          <widget class="QLabel" name="lbError">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>40</width>
             <height>60</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>40</width>
             <height>60</height>
            </size>
           </property>
           <property name="text">
            <string>故障
状态</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="lbImageError">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>70</width>
             <height>60</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>70</width>
             <height>60</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">border-image: url(:/Image/error.png);</string>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <property name="spacing">
          <number>5</number>
         </property>
         <item>
          <widget class="QLabel" name="lbAlarm">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>40</width>
             <height>60</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>40</width>
             <height>60</height>
            </size>
           </property>
           <property name="text">
            <string>报警
状态</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="lbImageAlarm">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>70</width>
             <height>60</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>70</width>
             <height>60</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">border-image: url(:/Image/alarm.png);</string>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_7">
       <property name="spacing">
        <number>10</number>
       </property>
       <item>
        <widget class="QToolButton" name="tBtnEdit">
         <property name="minimumSize">
          <size>
           <width>90</width>
           <height>60</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>90</width>
           <height>60</height>
          </size>
         </property>
         <property name="text">
          <string>编辑</string>
         </property>
         <property name="icon">
          <iconset resource="../image.qrc">
           <normaloff>:/Image/edit.png</normaloff>:/Image/edit.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>40</width>
           <height>30</height>
          </size>
         </property>
         <property name="toolButtonStyle">
          <enum>Qt::ToolButtonTextBesideIcon</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="tBtnSave">
         <property name="minimumSize">
          <size>
           <width>90</width>
           <height>60</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>90</width>
           <height>60</height>
          </size>
         </property>
         <property name="text">
          <string>保存</string>
         </property>
         <property name="icon">
          <iconset resource="../image.qrc">
           <normaloff>:/Image/save.png</normaloff>:/Image/save.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>40</width>
           <height>30</height>
          </size>
         </property>
         <property name="toolButtonStyle">
          <enum>Qt::ToolButtonTextBesideIcon</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="tBtnRestore">
         <property name="minimumSize">
          <size>
           <width>90</width>
           <height>60</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>90</width>
           <height>60</height>
          </size>
         </property>
         <property name="text">
          <string>恢复</string>
         </property>
         <property name="icon">
          <iconset resource="../image.qrc">
           <normaloff>:/Image/reset.png</normaloff>:/Image/reset.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>30</width>
           <height>30</height>
          </size>
         </property>
         <property name="toolButtonStyle">
          <enum>Qt::ToolButtonTextBesideIcon</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="tBtnZoomIn">
         <property name="minimumSize">
          <size>
           <width>90</width>
           <height>60</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>90</width>
           <height>60</height>
          </size>
         </property>
         <property name="text">
          <string>放大</string>
         </property>
         <property name="icon">
          <iconset resource="../image.qrc">
           <normaloff>:/Image/zoomin.png</normaloff>:/Image/zoomin.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>50</width>
           <height>50</height>
          </size>
         </property>
         <property name="toolButtonStyle">
          <enum>Qt::ToolButtonTextBesideIcon</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="tBtnZoomOut">
         <property name="minimumSize">
          <size>
           <width>90</width>
           <height>60</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>90</width>
           <height>60</height>
          </size>
         </property>
         <property name="text">
          <string>缩小</string>
         </property>
         <property name="icon">
          <iconset resource="../image.qrc">
           <normaloff>:/Image/zoomout.png</normaloff>:/Image/zoomout.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>50</width>
           <height>50</height>
          </size>
         </property>
         <property name="toolButtonStyle">
          <enum>Qt::ToolButtonTextBesideIcon</enum>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QWGraphicsView" name="graphicsView"/>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="spacing">
      <number>5</number>
     </property>
     <property name="leftMargin">
      <number>0</number>
     </property>
     <property name="rightMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="QGroupBox" name="groupBox">
       <property name="minimumSize">
        <size>
         <width>550</width>
         <height>130</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QScrollBar:vertical{
width:25px;
}</string>
       </property>
       <property name="title">
        <string>报警信息</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <property name="spacing">
         <number>0</number>
        </property>
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>5</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>5</number>
        </property>
        <item>
         <widget class="QTableWidget" name="tableWidgetAlarm">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>535</width>
            <height>95</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="verticalScrollBarPolicy">
           <enum>Qt::ScrollBarAlwaysOn</enum>
          </property>
          <property name="horizontalScrollBarPolicy">
           <enum>Qt::ScrollBarAlwaysOff</enum>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_2">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>550</width>
         <height>130</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QScrollBar:vertical{
width:25px;
}</string>
       </property>
       <property name="title">
        <string>故障信息</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <property name="spacing">
         <number>0</number>
        </property>
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>5</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>5</number>
        </property>
        <item>
         <widget class="QTableWidget" name="tableWidgetError">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>535</width>
            <height>95</height>
           </size>
          </property>
          <property name="verticalScrollBarPolicy">
           <enum>Qt::ScrollBarAlwaysOn</enum>
          </property>
          <property name="horizontalScrollBarPolicy">
           <enum>Qt::ScrollBarAlwaysOff</enum>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QWGraphicsView</class>
   <extends>QGraphicsView</extends>
   <header>ViewWidget/qwgraphicsview.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../image.qrc"/>
 </resources>
 <connections/>
</ui>
